### PR TITLE
Update other_samples.md to reflect current features/tech

### PR DESCRIPTION
**Pull request by pamelafox**
## Purpose

This PR updates the comparison to other samples document, based on what I can tell of the current state of the On Your Data repo.

## Does this introduce a breaking change?

When developers merge from main and run the server, azd up, or azd deploy, will this produce an error?
If you're not sure, try it out on an old environment.

```
[ ] Yes
[X] No
```

## Does this require changes to learn.microsoft.com docs?

This repository is referenced by [this tutorial](https://learn.microsoft.com/azure/developer/python/get-started-app-chat-template)
which includes deployment, settings and usage instructions. If text or screenshot need to change in the tutorial,
check the box below and notify the tutorial author. A Microsoft employee can do this for you if you're an external contributor.

```
[ ] Yes
[X] No
```

## Type of change

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[X] Documentation content changes
[ ] Other... Please describe:
```

## Code quality checklist

See [CONTRIBUTING.md](../CONTRIBUTING.md#submit-pr) for more details.

N/A

### CODE REVIEW

**Review by mattgotteiner — APPROVED** [review]

File: docs/data_ingestion.md [hunk 1]
```diff
@@ -11,6 +12,22 @@ This guide provides more details for using the `prepdocs` script to index docume
   - [Removal of documents](#removal-of-documents)
   - [Scheduled indexing](#scheduled-indexing)
 
+## Supported document formats
+
+In order to ingest a document format, we need a tool that can turn it into text. By default, use Azure Document Intelligence (DI in the table below), but we also have local parsers for several formats. The local parsers are not as sophisticated as Azure Document Intelligence, but they can be used to decrease charges.
+
+| Format | Manual indexing                      | Integrated Vectorization |
+| ------ | ------------------------------------ | ------------------------ |
```
Comment (pamelafox) [inline docs/data_ingestion.md:20]: @mattgotteiner Sorry, I ended up adding a section to a README about "Supported document formats" after sending this for review. What formats are supported by integrated vectorization? Is it truly this list here? https://learn.microsoft.com/en-us/azure/search/search-howto-indexing-azure-blob-storage#supported-document-formats

Comment (mattgotteiner) [reply docs/data_ingestion.md:20]: yes this is the correct list.

**Review by mattgotteiner — APPROVED** [review]

File: docs/data_ingestion.md [hunk 2]
```diff
@@ -11,6 +12,22 @@ This guide provides more details for using the `prepdocs` script to index docume
...
+| PDF    | Yes (DI or local with PyPDF)         | Yes                      |
+| HTML   | Yes (DI or local with BeautifulSoup) | Yes                      |
+| DOCX   | Yes (DI)                             | Yes                      |
+| PPTX   | Yes (DI)                             | Yes                      |
+| XLSX   | Yes (DI)                             | Yes                      |
+| Images (JPG, PNG, BPM, TIFF, HEIFF)| Yes (DI) | Yes                      |
+| TXT    | Yes (Local)                          | Yes                      |
+| JSON   | Yes (Local)                          | Yes                      |
+| CSV    | Yes (Local)                          | Yes                      |
+
```
Comment (pamelafox) [inline docs/data_ingestion.md:30]:
```suggestion
The Blob indexer used by the Integrated Vectorization approach also supports a few [additional formats](https://learn.microsoft.com/azure/search/search-howto-indexing-azure-blob-storage#supported-document-formats).

```

File: docs/data_ingestion.md [hunk 3]
```diff
@@ -11,6 +12,23 @@ This guide provides more details for using the `prepdocs` script to index docume
...
+| ------ | ------------------------------------ | ------------------------ |
+| PDF    | Yes (DI or local with PyPDF)         | Yes                      |
+| HTML   | Yes (DI or local with BeautifulSoup) | Yes                      |
+| DOCX   | Yes (DI)                             | Yes                      |
+| PPTX   | Yes (DI)                             | Yes                      |
+| XLSX   | Yes (DI)                             | Yes                      |
+| Images (JPG, PNG, BPM, TIFF, HEIFF)| Yes (DI) | Yes                      |
+| TXT    | Yes (Local)                          | Yes                      |
+| JSON   | Yes (Local)                          | Yes                      |
+| CSV    | Yes (Local)                          | Yes                      |
```
Comment (pamelafox) [inline docs/data_ingestion.md:29]:
```suggestion
| CSV    | Yes (Local)                          | Yes                      |

```

File: docs/data_ingestion.md [hunk 4]
```diff
@@ -11,6 +12,24 @@ This guide provides more details for using the `prepdocs` script to index docume
...
 
+## Supported document formats
+
+In order to ingest a document format, we need a tool that can turn it into text. By default, use Azure Document Intelligence (DI in the table below), but we also have local parsers for several formats. The local parsers are not as sophisticated as Azure Document Intelligence, but they can be used to decrease charges.
+
+| Format | Manual indexing                      | Integrated Vectorization |
+| ------ | ------------------------------------ | ------------------------ |
+| PDF    | Yes (DI or local with PyPDF)         | Yes                      |
+| HTML   | Yes (DI or local with BeautifulSoup) | Yes                      |
+| DOCX   | Yes (DI)                             | Yes                      |
```
Comment (pamelafox) [inline docs/data_ingestion.md:23]:
```suggestion
| DOCX, PPTX, XLSX   | Yes (DI)                             | Yes                      |
```

File: docs/data_ingestion.md [hunk 5]
```diff
@@ -11,6 +12,24 @@ This guide provides more details for using the `prepdocs` script to index docume
...
+| PPTX   | Yes (DI)                             | Yes                      |
+| XLSX   | Yes (DI)                             | Yes                      |
```
Comment (pamelafox) [inline docs/data_ingestion.md:25]:
```suggestion
```

File: docs/data_ingestion.md (hunk 1 above)
Comment (pamelafox) [reply docs/data_ingestion.md:20]: Great, adding link

**Pull request merged**